client/examples/register_1000_triggers.rs

```diff
@@ -75,15 +75,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let chain_id = get_chain_id();
     let genesis_key_pair = get_key_pair(test_network::Signatory::Genesis);
     let topology = vec![peer.id.clone()];
+    let genesis = generate_genesis(1_000_u32, chain_id.clone(), &genesis_key_pair, topology)?;
     let configuration = get_config(
         unique_vec![peer.id.clone()],
         chain_id.clone(),
         get_key_pair(test_network::Signatory::Peer),
-        genesis_key_pair.public_key(),
+        genesis.0.hash(),
```

Comment:
```suggestion
        genesis.hash(),
```

maybe make it private